README.md

```diff
@@ -290,6 +290,19 @@ DEBUG=1 ./build/teleport start -d
 Keep the server running in this mode, and make your UI changes in `/dist` directory.
 For instructions about how to update the Web UI, read [the `web` README](web#readme).
 
+### Pre-Commit Hooks
+
+This project supports pre-commit hooks. For download instructions and  more documentation visit the [pre-commit website](https://pre-commit.com/).
```

Comment:
I am not enthused by adding a python tool to our tooling. We currently do not use python for any tools, and it is a bit of a pain to manage the different versions (how many different "python environment manager" tools are there now? I've lost count and no longer know what I should be using).

Could we use something like lefthook instead: https://github.com/evilmartians/lefthook ? Written in Go, it's a trivial `go install` away from getting installed.

Reply:
Fair! I picked `pre-commit` because it's pretty standard and well supported. Honestly though it took a little bit to get it to work anyway because python. I updated it to use `lefthook`.